MovieMVVM/MovieMVVM/Services/CoreDataService/CoreDataServiceProtocol.swift

```diff
@@ -0,0 +1,12 @@
+// CoreDataServiceProtocol.swift
+// Copyright © Ilentiy. All rights reserved.
+
+import Foundation
+
+/// Протокол core data сервиса
```

Comment:
```suggestion
/// Протокол сервиса по сохранению данных в Core Data
```
я бы так назвал, коммент на вкус и цвет